tests/integration/kubernetes/k8s-confidential-attestation.bats

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bats
+# Copyright 2024 IBM Corporation
+# Copyright 2024 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../common.bash"
+load "${BATS_TEST_DIRNAME}/lib.sh"
+load "${BATS_TEST_DIRNAME}/confidential_common.sh"
+load "${BATS_TEST_DIRNAME}/confidential_kbs.sh"
+load "${BATS_TEST_DIRNAME}/tests_common.sh"
```

Comment:
`confidential_common.sh` includes `tests_common.sh` and `../../common.bash`. It's recommended to remove the lines `load "${BATS_TEST_DIRNAME}/../../common.bash"` and `load "${BATS_TEST_DIRNAME}/tests_common.sh"` to avoid redundancy.